signer/local/local.go

```diff
@@ -250,7 +253,19 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 		safeTemplate.SerialNumber = serialNumber
 	}
 
-	return s.sign(&safeTemplate, profile)
+	cert, err = s.sign(&safeTemplate, profile)
+
+	if profile.CTLog != "" {
+		var derCert, _ = pem.Decode(cert)
+		var ctclient = client.New(profile.CTLog)
+		chain := []ct.ASN1Cert{derCert.Bytes, s.ca.Raw}
+		var _, err = ctclient.AddChain(chain)
+		if err != nil {
+			return nil, err
```

Comment:
I think that the certificate should not be issued to the client until it's been submitted to the list of log servers specified in the config, which implies that it should fail if the submission fails.